liveblog.php

```diff
@@ -305,7 +306,13 @@ public static function handle_request() {
 			// we need to add the liveblog after the shortcodes are run, because we already
 			// process shortcodes in the comment contents and if we left any (like in the original content)
 			// we wouldn't like them to be processed
+
+			// Add Liveblog schema right before the content
+			add_filter( 'the_content', array( 'WPCOM_Liveblog_Schema', 'add_schema_to_content' ), 20);
```

Comment:
I'm really not sure that `the_content` is the best place for performing the schema generation. `the_content` can be called multiple times during a page load, eg.: during secondary loops. Shall we perhaps either put the schema to head of the document, or call the `WPCOM_Liveblog_Schema ::add_schema_to_content` directly from within the `WPCOM_Liveblog::add_liveblog_to_content` method?

Reply:
At the moment I'm adding the schema exactly where the Liveblog is being added as well. I didn't want to re-do all the logic inside `handle_request`. Adding it as a filter to the_content provided an option to remove it, if I nest it inside `add_liveblog_to-content` it's going to be sticked. That said, - it totally makes sense for the the schema to be printed directly from `add_schema_to_content` - I'll change that and just add a filter (in case someone doesn't want a schema or they have already built their own schema).